__tests__/app.test.js

```diff
@@ -138,4 +138,20 @@ describe("PATCH /api/reviews/:review_id", () => {
             expect(response.body.msg).toBe("Bad request")
         })
     })
+})
+
+describe("GET /api/users", () => {
+    test("GET:200 sends an array of user objects to the client", () => {
+        return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then((response) => {
+            const output = response.body.users
+            output.forEach((user) => {
```

Comment:
If you're going to check the array with a forEach we must assert that the array has a length of at least one